webclient/api/src/main/java/io/helidon/webclient/api/HttpClientConfigSupport.java

```diff
@@ -188,7 +188,8 @@ public void decorate(HttpClientConfig.BuilderBase<?, ?> target) {
             }
 
             if (target.proxy().isEmpty()) {
-                target.proxy(Proxy.create());
+                // Defaults to NoProxy type if Proxy is not set
+                target.proxy(Proxy.noProxy());
```

Comment:
Looks like this a departure from what was the default in prior versions which is NO_PROXY: 
https://github.com/helidon-io/helidon/blob/helidon-3.x/webclient/webclient/src/main/java/io/helidon/webclient/NettyClient.java#L48C64-L48C64
Was there a consensus decision to make this change? If so, do you know of a good way to determine that no proxy system properties was set?

Reply:
@jbescos Looks like this a departure from what was the default in prior versions which is NO_PROXY: 
https://github.com/helidon-io/helidon/blob/helidon-3.x/webclient/webclient/src/main/java/io/helidon/webclient/NettyClient.java#L48C64-L48C64
Was there a consensus decision to make this change? If so, do you know of a good way to determine that no proxy system properties was set?

Reply:
There is one issue related to make it as default:
https://github.com/helidon-io/helidon/issues/7274

I don't know if this is exactly what you need, but to determine whether the system properties are not set, or they are set but it does not apply (because the host is in the noHosts system property), you can:
```
List<java.net.Proxy> proxies = proxy.systemProxySelector
                        .select(URI.create(scheme + "://" + targetAddress.getHostName() + ":" + targetAddress.getPort()));
proxies.isEmpty()
```
But note you need to know in advance the URI that you want to request

Reply:
@jbescos, please recheck the recent changes. Default is back to System Proxy and System Proxy when used is added in the relativeUris config logic.